sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryServicesImpl.java

```diff
@@ -401,11 +401,9 @@ public Job getJob(JobReference jobRef, Sleeper sleeper, BackOff backoff)
     private static final FluentBackoff INSERT_BACKOFF_FACTORY =
         FluentBackoff.DEFAULT.withInitialBackoff(Duration.millis(200)).withMaxRetries(5);
 
-    // A backoff for rate limit exceeded errors. Retries forever.
+    // A backoff for rate limit exceeded errors. Only retry upto approximately 2 minutes.
```

Comment:
Probably add a comment as to why we are limiting this.

Reply:
Done.